exercises/nth-prime/nth_prime_test.rb

```diff
@@ -22,13 +25,35 @@ def test_sixth_prime
 
   def test_big_prime
     skip
-    assert_equal 104_743, Prime.nth(10_001)
+    assert_equal 104743, Prime.nth(10001)
   end
 
-  def test_weird_case
+  def test_there_is_no_zeroth_prime
     skip
     assert_raises ArgumentError do
       Prime.nth(0)
     end
   end
```

Comment:
I think raising an ArgumentError is ok.

The important thing is that the generator generates a test file that doesn't need further modification, so if/when the .json file changes in future, we can just re-generate the test file and not worry about any special cases lurking in there that need manual tweaking.